### DISCUSSION TITLE
AdguardHome pFsense installation and configuration

### DISCUSSION THREAD

**Discussion by Adgbeta** [AdguardTeam/AdGuardHome#6704] [Q&A]
What is the best way to install AdguardHome in pFsense and how to configure pFsense DNS resolver?

I tried to force all devices to use encrypted DNS, but doesn't work always Adguardhome use plain DNS, when uncheck "Enable plain DNS" to force encryption, Adguarhome stop resolving DNS and I lost internet connection.

Any Ideas?

Thanks

**Comment by Fredoah** [comment]
Hi !
https://amefs.net/en/archives/2645.html
